<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-GHSE0P0" name="getRequest.GetData" tests="5" failures="1" timestamp="25 Jan 2019 23:15:45 GMT" time="76.331" errors="0">
  <testcase name="given_pageParameter_when_jumpOnPageParameter_then_showFirstNameOnList" time="0.267" classname="getRequest.GetData">
    <failure type="org.json.JSONException" message="JSONObject[&quot;items&quot;] not found.">
      <![CDATA[org.json.JSONException: JSONObject["items"] not found.
	at org.json.JSONObject.get(JSONObject.java:471)
	at org.json.JSONObject.getJSONArray(JSONObject.java:618)
	at getRequest.GetData.given_pageParameter_when_jumpOnPageParameter_then_showFirstNameOnList(GetData.java:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- given_pageParameter_when_jumpOnPageParameter_then_showFirstNameOnList -->
  <testcase name="given_listmorethan10_when_exceededTheRateLimit_then_forbiddenResponseisThrown" time="71.297" classname="getRequest.GetData"/>
  <testcase name="given_RepositoriesofTopicExists_when_RepositoryInfoIsRetrieved_then_higestFolks" time="1.621" classname="getRequest.GetData"/>
  <testcase name="given_UserOrTopicExists_when_UserInfoIsRetrieved_then_200StatusCodeIsReceivedWithCount" time="2.387" classname="getRequest.GetData"/>
  <testcase name="given_className_when_langAndFile_then_showRepo" time="0.759" classname="getRequest.GetData"/>
</testsuite> <!-- getRequest.GetData -->
